### PR TITLE
version to 0.10.0

### PR DESCRIPTION
**Pull request by scottdraves**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
So the commit added is a re-rendering. It cut the Windows 32-bit builds and Python 2 builds because of [these `skip` lines]( https://github.com/conda-forge/beakerx-feedstock/blob/d7c99227e800d31db17e9c5a5a6afd6465ed522f/recipe/meta.yaml#L20-L21 ). While the builds can be manually added back to the CIs, they are not producing packages currently and have not been for some time, which is a suboptimal use of the orgs resources. If those are meant to be built, would recommend dropping the `skip` lines and then re-rendering. Otherwise it would be good not to run builds that do not produce packages.

**Comment by scottdraves** [comment]
thanks. the changes seemed ok but i didn't see that you triggered them, and they happened at the same time as our PR,  because i was looking at "git log" not here, and i didn't see them on https://github.com/conda-forge/beakerx-feedstock/commits/master, sorry about that.  will put them back.

**Comment by jakirkham** [comment]
No worries. This is part of a [new webservice]( https://conda-forge.org/docs/webservice.html ) that we have been trying out of late, which you are welcome to use as well. If you have any suggestions on how to make things more clear, happy to integrate them into the procedure.

**Comment by jakirkham** [comment]
On a different note, it seems like you are running into [an issue that I encountered earlier today]( https://github.com/conda-forge/jupyter_client-feedstock/pull/14#issuecomment-352510987 ) (e.g. [this build]( https://ci.appveyor.com/project/conda-forge/beakerx-feedstock/build/1.0.42/job/t78a32sqdr2xoh7l#L12991 )). Might try installing with `pip` instead to see if that resolves the issue.

**Comment by jmsdnns** [comment]
Interesting. Your issue looks exactly the same. Will look into that. Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,10 +16,10 @@ source:
   sha256: {{ sha256 }}
   
 build:
-  number: 0
+  number: 1
   skip: true  # [py2k]
   skip: true  # [win32]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install .
```
Comment (jakirkham) [inline recipe/meta.yaml]: Will need the `--no-deps` argument as well.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -16,10 +16,10 @@ source:
   sha256: {{ sha256 }}
   
 build:
-  number: 0
+  number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: This can stay `0` as we are doing a new release and there are no broken builds that we are fixing yet.

**Pull request merged**